README.md

```diff
@@ -23,15 +25,58 @@ Terraform 0.12. Pin module version to ~> 2.0. Submit pull-requests to master bra
 
 Terraform 0.11. Pin module version to ~> 1.0. Submit pull-requests to terraform011 branch.
 
-## Usage
-
-### Put an example usage of the module here
+### Usage for combined policy statements
 
 ```hcl
-module "example" {
-  source = "terraform/registry/path"
+// TODO: make sure these references to module names are correct once pushed to the registry
+module "ou_scp" {
+  source = "trussworks/ou-scp"
+  target = {
+    name = "my_ou"
+    id = aws_organizations_organizational_unit.my_ou.id
+  }
```

Comment:
This should just look like this:
```suggestion
  target = aws_organizations_organizational_unit.my_ou
```

Your target is the actual OU resource, the variable definition just pulls those attributes we added in `variables.tf` from the object directly.